index.html

```diff
@@ -1961,6 +1961,48 @@ <h3>Layer Cryptographic Options Appropriately</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Agility vs. Hardening</h3>
+
+        <p>
+Cryptographic agility is a practice where one designs
+<em>frequently connected</em> information security systems to <em>support</em>
+multiple cryptographic primitives and algorithms at the same time. The primary
+goal of cryptographic agility is to enable systems to rapidly adapt to new
+cryptographic primitives and algorithms without making disruptive changes to the
+systems' infrastructure. Thus, when a particular cryptographic primitive is
+determined to be no longer safe to use, such as the SHA-1 algorithm, systems can
+be reconfigured to use a newer primitive via a simple configuration file change.
+        </p>
+        <p>
+Cryptographic agility is often effective when both the client and the server in
+the information security system are in regular contact. However, when the
+messages protected by a particular cryptographic algorithm are long-lived, like
+they are in Verifiable Credentials, and when the client (holder) might not be
+able to easily recontact the server (issuer), then cryptographic agility does
+not provide the protections necessary.
+        </p>
+        <p>
+<dfn class="lint-ignore">Cryptographic hardening</dfn> is a practice where one
```

Comment:
This has a &lt;dfn&gt; HTML tag, whereas "Cryptographic agility" above doesn't? Otherwise looks good.

Reply:
Good catch, I have added the missing dfn to "Cryptographic agility".